command/assets/connect.nomad

```diff
@@ -0,0 +1,467 @@
+      driver = "docker"
+
+      # The "config" stanza specifies the driver configuration, which is passed
+      # directly to the driver to start the task. The details of configurations
+      # are specific to each driver, so please see specific driver
+      # documentation for more information.
+      config {
+        image = "hashicorpnomad/counter-api:v1"
+      }
+
+      # The "artifact" stanza instructs Nomad to download an artifact from a
+      # remote source prior to starting the task. This provides a convenient
+      # mechanism for downloading configuration files or data needed to run the
+      # task. It is possible to specify the "artifact" stanza multiple times to
+      # download multiple artifacts.
+      #
+      # For more information and examples on the "artifact" stanza, please see
+      # the online documentation at:
+      #
+      #     https://www.nomadproject.io/docs/job-specification/artifact.html
+      #
+      # artifact {
+      #   source = "http://foo.com/artifact.tar.gz"
+      #   options {
+      #     checksum = "md5:c4aa853ad2215426eb7d70a21922e794"
+      #   }
+      # }
+
+
+      # The "logs" stanza instructs the Nomad client on how many log files and
+      # the maximum size of those logs files to retain. Logging is enabled by
+      # default, but the "logs" stanza allows for finer-grained control over
+      # the log rotation and storage configuration.
+      #
+      # For more information and examples on the "logs" stanza, please see
+      # the online documentation at:
+      #
+      #     https://www.nomadproject.io/docs/job-specification/logs.html
+      #
+      # logs {
+      #   max_files     = 10
+      #   max_file_size = 15
+      # }
+
+      # The "resources" stanza describes the requirements a task needs to
+      # execute. Resource requirements include memory, network, cpu, and more.
+      # This ensures the task will execute on a machine that contains enough
+      # resource capacity.
+      #
+      # For more information and examples on the "resources" stanza, please see
+      # the online documentation at:
+      #
+      #     https://www.nomadproject.io/docs/job-specification/resources.html
+      #
+      resources {
+        cpu    = 500 # 500 MHz
+        memory = 256 # 256MB
+      }
+    }
+
+    # The Envoy sidecar admission controller will inject an Envoy task into
+    # any task group for each service with a sidecar_service stanza it contains.
+    # A group network will be required and a dynamic port will be registered for
+    # remote services to connect to Envoy with the name `connect-proxy-<service>`.
+    # By default, Envoy will be run via its official upstream Docker image.
+    #
+    # There are two ways to modify the default behavior:
+    #   * Tasks can define a `sidecar_task` stanza in the `connect` stanza
+    #     that merges into the default sidecar configuration.
+    #   * Tasks can use the `sidecar_service` stanza to refer to a task
```

Comment:
You don't need to refer to it in the sidecar_service stanza. You just need to make sure the kind is set correctly.

Reply:
Oh, because the `kind = "connect:<service>"` already includes the service name to pair them up?

Reply:
Yup!

Reply:
Done.